descholar-ceo/AnnounceIT
# #170727073 user sees announcements by state

#### What does this PR ​ do​ ?
As a user, I should be to view a specific announcement by entering id or state of an announcement

#### Description of Task to be completed?
-User enters an id number of an announcement in a search bar
-User gets all announcements having state or id related to what he/she typed

#### How should this be manually tested?
* Clone this repository in case you don't have it using this command `git clone https://github.com/descholar-ceo/AnnounceIT.git`
* If you use [VSCode](https://code.visualstudio.com/) as code editor, install [_live-server_](https://marketplace.visualstudio.com/items?itemName=ritwickdey.LiveServer), and click on _Go Live_

* Click on enter button from homepage, 

* And then you will be redirected in the browser, here you will need to go to the root of the url you have in url-bar, and enter like `http://127.0.0.1:5500/UI/html/user-profile.html`, you will be redirected to the users page, or enter like this ``http://127.0.0.1:5500/UI/html/user-profile.html`,
* Enter any number you want or try with words like `active, deactivated, accepted, declined` in search bar, you will get results as you type

* Or if you don't use [VSCode](https://code.visualstudio.com/) as text editor, after cloning the repo, you navigate in `AnnounceIT/UI/html/` and you will see different files, you choose `user-profile.html` and right click and open with your preferred web browser

#### Any background context you want to provide?
* It would be better if you use  [VSCode](https://code.visualstudio.com/)

#### What are the relevant pivotal tracker stories?
[170727073](https://www.pivotaltracker.com/n/projects/2429498#)

#### Screenshots (​ if​ appropriate)
![Screenshot from 2020-01-20 01-56-07](https://user-images.githubusercontent.com/56886064/72690756-124d2e00-3b28-11ea-9063-ab7e8b33c42a.png)


#### Questions:
No Question
